HFuse.cabal

```diff
@@ -34,6 +34,7 @@ Library
   else
       if os(freebsd)
          Includes:           sys/param.h, sys/mount.h
+         CC-Options:           "-DFUSE_USE_VERSION=26"
```

Comment:
I think if it is os specific parameter then it should be under `if os(...)` and may be also with `if os-version(>=..) restriction.

Reply:
The `if os(freebsd)` conditional already exists on line 35. Currently we have FreeBSD versions **11.2** and **12.0** supported and these releases are not different in this way so I think that using `if os-version(...)` is not necessary in case of this change.